Remotion/Web/Development.WebTesting/Configuration/TestInfrastructureConfiguration.cs

```diff
@@ -38,20 +38,39 @@ public class TestInfrastructureConfiguration : ITestInfrastructureConfiguration
     private readonly string _screenshotDirectory;
     private readonly TimeSpan _searchTimeout;
     private readonly TimeSpan _retryInterval;
+    private readonly TimeSpan _commandTimeout;
     private readonly IRequestErrorDetectionStrategy _requestErrorDetectionStrategy;
     private readonly bool _closeBrowserWindowsOnSetUpAndTearDown;
 
     public TestInfrastructureConfiguration ([NotNull] WebTestConfigurationSection webTestConfigurationSection)
+        : this (
+            webTestConfigurationSection.WebApplicationRoot,
+            webTestConfigurationSection.ScreenshotDirectory,
+            webTestConfigurationSection.SearchTimeout,
+            webTestConfigurationSection.RetryInterval,
+            webTestConfigurationSection.CommandTimeout,
+            webTestConfigurationSection.CloseBrowserWindowsOnSetUpAndTearDown,
+            GetRequestErrorDetectionConfiguration (webTestConfigurationSection.RequestErrorDetectionStrategyTypeName))
     {
       ArgumentUtility.CheckNotNull ("webTestConfigurationSection", webTestConfigurationSection);

```

Comment:
We can drop this check since the ctor's body is empty. If we do want this check to be helpful, it should be added to the first parameter of the next-call in line 47.